<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fwtestClass</class>
 <widget class="QMainWindow" name="fwtestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fwtest</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Un_Subscribe">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsubscribe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subscribe">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribe</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Communicate With Packet Tracer</string>
    </property>
    <widget class="QLineEdit" name="ledAddress">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledPort">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>39000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Start_Framework">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string comment="Framework" extracomment="Start Framework"/>
     </property>
     <property name="whatsThis">
      <string extracomment="Start Framework"/>
     </property>
     <property name="accessibleName">
      <string extracomment="Start Framework"/>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Start Framework1"/>
     </property>
     <property name="text">
      <string>Start ...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledAddress_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="workspaceCLIO">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>166</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Curtin University of Technology</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string>background-image: url(../newPrefix/curtin.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="devCLIO">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>521</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Logical Workspace Components</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="workspaceCLI">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Events">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Workspace Management</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>291</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Device Management:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="devCLI">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Events:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Communicate With Mac OS</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="udpPort">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>7777</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startUDP">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start ...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Qt/2009.01/qt/myimages.qrc"/>
 </resources>
 <connections/>
</ui>
